string litt1
string shape1
string litt2
string shape2
---
string mat3
